tests/Feature/Api/PublicScheduleTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Tests\Feature\Api;
+
+use App\Models\Lecturing;
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class PublicScheduleTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /** @test */
+    public function can_get_schedules_within_specified_date_range() {
+        $lecturing = factory(Lecturing::class)->create([
+            'audience_code' => Lecturing::AUDIENCE_FRIDAY,
+            'date' => Carbon::tomorrow()->format('Y-m-d')
```

Comment:
Kita pakai ini untuk array ya biar seragam sama convetion kita terapkan:
https://docs.styleci.io/fixers#trailing_comma_in_multiline_array
> Multiline arrays MUST have a trailing comma.
```suggestion
            'date' => Carbon::tomorrow()->format('Y-m-d'),
```